spec/outputs/neo4j_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+require "logstash/plugin"
+require "logstash/json"
+
+describe "outputs/neo4j" do
+
+  context "registration" do
+
+    let(:output) { LogStash::Plugin.lookup("output", "neo4j").new("path" => "/tmp/db") }
+
+    it "should register" do
+      expect {output.register}.to_not raise_error
+    end
+
+    it "should teardown" do
+      output.register
+      expect {output.teardown}.to_not raise_error
+    end
```

Comment:
To be consistent in the style of this file I would add space after { and before }, all the other procs are defined like this.